src/Rector/Contrib/PHPUnit/SpecificMethod/AssertPropertyExistsRector.php

```diff
@@ -74,11 +76,13 @@ public function isCandidate(Node $node): bool
     }
 
     /**
-     * @var MethodCall
+     * @param MethodCall $methodCallNode
      */
```

Comment:
Here it's correct

Reply:
Do you know about Sniff/Fixer that could check that?

Reply:
I'll search for, but AFAIK: no :(